src/library_gl.js

```diff
@@ -822,27 +821,32 @@ var LibraryGL = {
 #endif
 
       // Detect the presence of a few extensions manually, this GL interop layer itself will need to know if they exist. 
+#if LEGACY_GL_EMULATION
       context.compressionExt = GLctx.getExtension('WEBGL_compressed_texture_s3tc');
       context.anisotropicExt = GLctx.getExtension('EXT_texture_filter_anisotropic');
-      context.floatExt = GLctx.getExtension('OES_texture_float');
```

Comment:
Oh ok, I was confused because it was removed but not otherwise referenced in this diff. So it was referenced twice before, this just removes a duplication.